sdrgui/mainwindow.h

```diff
@@ -229,7 +229,7 @@ private slots:
     void openDeviceSetPresetsDialog(QPoint p, const DeviceGUI *deviceGUI);
 	void commandKeyPressed(Qt::Key key, Qt::KeyboardModifiers keyModifiers, bool release) const;
 	void fftWisdomProcessFinished(int exitCode, QProcess::ExitStatus exitStatus);
-    void orientationChanged(Qt::ScreenOrientation orientation) const;
+    void orientationChanged(Qt::ScreenOrientation orientation);
```

Comment:
Any valid reason to remove const ?

Reply:
Yes, "setTabPosition" ,ethod is non-const, so calling it from const method isn't correct.